source/_posts/2020-03-18-subspace-2.0.md

```diff
@@ -0,0 +1,158 @@
+title: Subspace 2.0
+author: richard_ramos
+summary: "New components to simplify the usage of Subspace within React projects"
+categories:
+  - announcements
+  - releases
+  - subspace
+layout: blog-post
+image: '/assets/images/subspace.png'
+---
+
+Subspace 2.0
+===
+
+New to subspace? Check out the website at [http://subspace.embarklabs.io/](http://subspace.embarklabs.io/)
+
+## Introducing Subspace React's components
+A new set of components were created to simplify the usage of Subspace within React projects. Learn more about them here https://subspace.embarklabs.io/react.html
+
+### Install
+You can install it through `npm` or `yarn`:
+```
+npm install --save @embarklabs/subspace-react web3 rxjs
+```
+
+
+### Usage
+
+#### SubspaceProvider
+To use most of the `subspace-react` components, you need to wrap your app with the `<SubspaceProvider web3={web3} />` component. This will make Subspace available to any nested components that accesses it via the `useSubspace` hook or has been wrapped in the `withSubspace` higher order component. Any React component might use Subspace so it makes sense to add the provider near the top level of your dApp. The `SubspaceProvider` requires a web3 object
+
+```js
+// index.js
+import React from 'react'
+import ReactDOM from 'react-dom'
+import MyApp from './MyApp'
+import { SubspaceProvider } from '@embarklabs/subspace-react';
+
+const web3 = new Web3("ws://localhost:8545");
+
+const rootElement = document.getElementById('root')
+ReactDOM.render(
+  <SubspaceProvider web3={web3}>
+    <MyApp />
+  </SubspaceProvider>,
+  rootElement
+);
+```
+
+#### useSubspace
+Rather than relying on global variables or passing Subspace through props, The easiest way to access Subspace features is via the `useSubspace` hook. Be sure that your entire dApp is wrapped with a `<SubspaceProvider />` to have it available througout the component tree.
```

Comment:
```suggestion
Rather than relying on global variables or passing Subspace through props, the easiest way to access Subspace features is via the `useSubspace` hook. Be sure that your entire DApp is wrapped with a `<SubspaceProvider />` to have it available throughout the component tree.
```